app/src/main/java/com/checkin/app/checkin/Home/GetDayDate.java

```diff
@@ -0,0 +1,81 @@
+package com.checkin.app.checkin.Home;
+
+import android.util.Log;
+
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.util.ArrayList;
+import java.util.Calendar;
+import java.util.Date;
+
+public class GetDayDate {
+
+    public static ArrayList<String>getFiveDays(){
+        ArrayList<String>tosend=new ArrayList<>();
+
+        for (int i = 0; i <5 ; i++) {
+            Calendar calendar=Calendar.getInstance();
```

Comment:
Could reuse calendar instance and just add  1 at each iteration if you are operation on the same instance.